src/m365/spo/commands/user/user-get.spec.ts

```diff
@@ -212,6 +308,145 @@ describe(commands.USER_GET, () => {
     }));
   });
 
+  it('retrieves user by userName with output option json', async () => {
+    sinon.stub(request, 'get').callsFake(async (opts) => {
+      if (opts.url === `${validWebUrl}/_api/web/siteusers?$filter=UserPrincipalName eq ('${formatting.encodeQueryParameter(validUserName)}')`) {
+        return {
+          "value": [userResponse]
+        };
+      }
+
+      if ((opts.url as string).indexOf('/_api/web/siteusers/GetById') > -1) {
+        return {
+          "value": [userResponse]
+        };
```

Comment:
If I'm not mistaken, the response will not be an array, but should be just a single value